.gitignore

```diff
@@ -19,10 +19,11 @@ chromedriver*
 *.db
 *.md.html
 libs/
-*.js
-!webpack-*.config.js
+src/main/webapp/js
+src/main/webapp/test/*.js
 node_modules/
 npm-debug.log
+package-lock.json
```

Comment:
For a moment I thought you introduced `package-lock.json` lol. Is this intended?

Reply:
We have always been. Only this time I want to make it explicit.